src/sentry/incidents/endpoints/organization_alert_rule_index.py

```diff
@@ -167,7 +167,7 @@ def get(self, request: Request, organization) -> Response:
             # Filter to only error alert rules
             alert_rules = alert_rules.filter(snuba_query__dataset=Dataset.Events.value)
         issue_rules = Rule.objects.filter(
-            status=ObjectStatus.ACTIVE,
+            status__in=[ObjectStatus.ACTIVE, ObjectStatus.DISABLED],
```

Comment:
can we add a test to confirm this behavior?

Reply:
Added a test 👍

Reply:
thanks!